tests/unit/test_analyzer_util_suite.py

```diff
@@ -353,3 +352,38 @@ def test_join_statement_negative():
         ValueError, match="A join should either have using clause or a join condition"
     ):
         join_statement("", "", join_type, "cond2", "", False)
+
+
+def test_create_iceberg_table_statement():
+    assert create_table_statement(
+        table_name="test_table",
+        schema="test_col varchar",
+        is_iceberg=True,
+        external_volume="example_volume",
+        catalog="example_catalog",
+        base_location="/root",
+        catalog_sync="integration_name",
+        storage_serialization_policy="OPTIMIZED",
+    ) == (
+        " CREATE    ICEBERG  TABLE test_table(test_col varchar)  EXTERNAL_VOLUME  = 'example_volume' "
+        " CATALOG  = 'example_catalog'  BASE_LOCATION  = '/root'  CATALOG_SYNC  = 'integration_name'"
+        "  STORAGE_SERIALIZATION_POLICY  = 'OPTIMIZED' "
+    )
+
+
+def test_create_iceberg_table_as_select_statement():
```

Comment:
can we try to see if we can add a simple integration test as well. Maybe use ddl statement to check if the properties are set correctly? Might be harder since we are dealing with iceberg tables.